Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/audit/transform/DocRetrieveAuditTransforms.java

```diff
@@ -410,9 +413,9 @@ protected ActiveParticipant getActiveParticipantDestination(NhinTargetSystemType
 
         participant.getRoleIDCode()
             .add(AuditDataTransformHelper.createCodeValueType(
-                    AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DEST, null,
-                    AuditTransformsConstants.ACTIVE_PARTICIPANT_CODE_SYSTEM_NAME,
-                    AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DESTINATION_DISPLAY_NAME));
+                AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DEST, null,
+                AuditTransformsConstants.ACTIVE_PARTICIPANT_CODE_SYSTEM_NAME,
+                AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DESTINATION_DISPLAY_NAME));
         return participant;
     }
```

Comment:
Can you refactor this method to refactor to set TypeValuePairType into a separate method  getParticipantObjectDetail(byte[] repositoryUniqueId, byte[] homeCommunityId, ParticipantObjectIdentificationType participantObject) ?